src/frontend/e2e/pages/filter.ts

```diff
@@ -1,103 +1,101 @@
 import { Page } from "@playwright/test";
-export abstract class FilterSample {
-  public static async applyFilter(
-    page: Page,
-    filterData: Partial<FilterData>
-  ): Promise<void> {
-    // wait for sample page is ready to be handled
-    await page.waitForFunction(() => {
-      const samples = document.querySelector(
-        "a[href$='/data/samples'] > div > div:nth-child(2)"
-      )?.textContent;
-      return parseInt(samples!) > 0;
-    });
-    if (
-      filterData.uploadDateFrom !== undefined ||
-      filterData.uploadDateTo !== undefined
-    ) {
-      await page.locator("button[label='Upload Date']").click();
-      if (filterData.uploadDateFrom !== undefined) {
-        await page
-          .locator("input[name='uploadDateStart']")
-          .fill(filterData.uploadDateFrom);
-      }
-      if (filterData.uploadDateTo !== undefined) {
-        await page
-          .locator("input[name='uploadDateEnd']")
-          .fill(filterData.uploadDateTo);
-      }
+
+const applyCollectionDateSelector =
+  "​to​ApplyLast 7 DaysLast 30 DaysLast 3 MonthsLast 6 MonthsLast Year";
```

Comment:
Is there a reason these need to be joined without spaces? If so, a comment might be good.  If I saw this in the code, I would be tempted to "fix" it.

Reply:
I have added comments. Hopefully we will replace them with data-test-id, but for now this is how PL sees the date options in dropdown